src/pages/productDetailsPage/ProductDetails.jsx

```diff
@@ -125,11 +159,19 @@ const ProductDetails = () => {
   // navigate is used by react router
   const navigate = useNavigate();
 
-  const { isDesktop } = useRecoilValue(windowSize);
+  const { isDesktop, mobile } = useRecoilValue(windowSize);
 
   // Get hit attribute from config file
-  const { image, productName, brand, sizeFilter, colour, colourHexa } =
-    hitsConfig;
+  const {
```

Comment:
what happens if the SE isn't using one of these attributes because they don't have the data for example colourHexa?

Reply:
@VirAlex

Reply:
There is already a const `hexaCode` line 158 to check is there is hexacode or not